var/spack/repos/builtin/packages/votca/package.py

```diff
@@ -34,12 +35,12 @@ class Votca(CMakePackage):
     depends_on("eigen@3.3:")
     depends_on("boost")
     depends_on('mkl', when='+mkl')
-    depends_on("libxc")
+    depends_on("libxc", when='+xtp')
     depends_on("hdf5+cxx~mpi")
-    depends_on("libint@2.6.0:")
-    depends_on("libecpint")
-    depends_on("py-h5py")
-    depends_on("py-lxml")
+    depends_on("libint@2.6.0:", when='+xtp')
+    depends_on("libecpint", when='+xtp')
+    depends_on("py-h5py", when='+xtp')
+    depends_on("py-lxml", when='+xtp')
```

Comment:
Can you put all the xtp dependencies inside a `with when("+xtp"):` block?

Reply:
Done